### DISCUSSION TITLE
Support for polling

### DISCUSSION THREAD

**Discussion by webysther** [ptr727/PlexCleaner#285] [Ideas]
There a chance to support polling as an option for monitor? like `monitor --polling=30s`.

**Comment by ptr727** [comment]
That would be equivalent to running `process` in a loop with a 30s wait, and not very efficient as it would probably (depends on collection size) take much longer to process than 30s.  
What is the issue with `monitor`?

**Comment by webysther** [comment]
Good one @ptr727, I don't think to run in loop before. Normally people using docker create a direct mount to disk, but if you have a NAS (Synology/QNAP) you create a named volume with nfs for instance, this enable 'scale' with docker in a small home, in my case, NAS + 2 mac mini with linux. Almost every software has a strategy with work, but when we talk fs-watch this become a problem, only calibre with fs watch don't work and I have +50 docker compose projects running. VS Code docker uses polling like prometheus and others. The ideia are not good but is more compatible with more deploy strategies. In my case, I want to use to monitor a folder with downloaded video which can be done with monitor. But if I have polling this become a plug-and-lay solution, maybe for others like me.
